### PR TITLE
Fixing Pipe Bombs

### LINKED ISSUE #26778: Radio Pipe Bombs are significantly stronger than their non-radio versions.
**Describe the Issue**  
Despite using the same amount of explosives, radio Pipe bombs are set to have absurd amounts of power, range and shrapnel.

**Expected behavior**  
Given that the Radio pipe bomb only adds some electronic bits and a pilot light to ignite, the discrepancy is odd.  They should have the same power and range.  Just a different activation method.  Dynamite and Radio Dynamite have the same power.

**Screenshots**  
Normal Pipe bomb: 
image

Radio Activated Pipe Bomb: 
image

### PR DESCRIPTION
**Pull request by Xion350**
#### Summary
`SUMMARY: Bugfixes "Removes Radio Pipe Bombs, Radio Activated Dynamite and Fixes Pipe bombs"`

#### Purpose of change
Pipe Bombs and their Radio Activated Counterparts had very strange discrepancies as described in #26778 Changing weight, power and function.

#### Describe the solution
Pipe Bombs previously used obsolete code.  This PR fixes #26778 by adding the `explosion` use to Pipe Bombs rather than hard code.  To clean things up, they are also valid targets for the Radio Activation Mod, making the Radio Activated versions pointless as the Radio activation mod costs the same to make and attach.  They are functionally identical modded or created as a Radio Activated Pipe Bomb.

Radio Activated Dynamite was also removed because you can attach the Radio Activation Mod to it anyway.

The killing power of the new Pipe Bombs are fairly realistic, killing normal zeds with almost certainty in adjacent tiles, but the shrapnel lacks the power to kill beyond that.

#### Describe alternatives you've considered
The hard code referred to in `"use_action": "PIPEBOMB_ACT"` could likely be removed with no problem, but I'll let someone else decide if that's necessary or not.  The chance for Pipe Bombs to fizzle out had to be removed as I don't think that effect can be done through .json.

#### Additional context
As this is my first PR, I likely have made many mistakes.  But this is a fairly simple edit so it should be okay.  I'm also fine with someone else taking this idea and doing it better.

### CODE REVIEW

File: data/json/items/tools.json [hunk 1]
```diff
@@ -5495,54 +5445,18 @@
...
       "type": "explosion",
-      "no_deactivate_msg": "You can't disarm your %s.  Place it, get away, and use the remote control to detonate.",
-      "explosion": { "power": 500, "shrapnel": { "casing_mass": 500, "fragment_mass": 1.0 } }
-    },
-    "flags": [ "RADIO_ACTIVATION", "RADIOSIGNAL_1", "RADIOCARITEM", "BOMB", "TRADER_AVOID" ]
+      "sound_volume": 0,
+      "sound_msg": "sss...",
+      "no_deactivate_msg": "You've already lit the %s, try throwing it instead.",
+      "explosion": { 
+	  "power": 20,
```
Comment (Night-Pryanik) [inline data/json/items/tools.json]: Replace tabs with spaces here and below.

**Comment by Night-Pryanik** [comment]
> The hard code referred to in "use_action": "PIPEBOMB_ACT" could likely be removed with no problem, but I'll let someone else decide if that's necessary or not.

Delete it if it isn't used anywhere in the code or json.

**Comment by Night-Pryanik** [comment]
Also, please use linter (http://dev.narc.ro/cataclysm/format.html) to properly format your changes.

**Comment by Xion350** [comment]
Removed the 3 instances of Hardcoded `PIPEBOMB_ACT`, the other references in json are referring to the item.

**Comment by Xion350** [comment]
Everything should be in order now.

**Pull request merged**